execute_wsh_dispatch.h

```diff
@@ -81,6 +90,12 @@ static HRESULT STDMETHODCALLTYPE script_dispatch_get_ids_of_names(IDispatch *dis
 static HRESULT STDMETHODCALLTYPE script_dispatch_invoke(IDispatch *dispatch, DISPID disp_id, REFIID riid, LCID lcid,
                                                         WORD flags, DISPPARAMS *disp_params, VARIANT *result_ptr,
                                                         EXCEPINFO *excep_info, UINT *arg_err) {
+    UNUSED(dispatch);
+    UNUSED(riid);
+    UNUSED(lcid);
+    UNUSED(flags);
+    UNUSED(excep_info);
+    UNUSED(arg_err);
```

Comment:
6 almost empty lines.
Why I did not get warning here when enabled /W4?

Reply:
I am not trying to optimize for shortest number of lines. I want straight-forward code.

Reply:
Ans build errors because of `#ifdef _DEBUG` that definitely makes code with hollows.

Reply:
`UNUSED(lcid, flags, excep_info, arg_err);` is complex??? More than ` UNUSED(lcid);
    UNUSED(flags);
    UNUSED(excep_info);
    UNUSED(arg_err);`???

Reply:
All the `UNUSED` macro scaffolding is too much for such a simple thing. It is overkill.

Reply:
I have removed the `id` variable altogether. It is the same as what is done in threadpool_pthread.c `LOG_DEBUG` calls.